components/react/hooks/README.md

```diff
@@ -451,3 +451,42 @@ const Component = ({value, initialValue, onClick, onReset}) => {
   )
 }
 ```
+
+### useDynamicHeightCollapsible
+
+> The `useDynamicHeightCollapsible` hook has the purpose of controlling the MoleculeCollapsible component when you need its height to be calculated dynamicaly.
+
+```js
+import {useRef} from 'react'
+
+const containerListRef = useRef()
+
+const getContentHeight = (
```

Comment:
Where are the properties for the `getContentHeight` function coming from? I see you are passing its reference to the hook, which does not provide those props

Reply:
it is just an example, they are defined directly from the Component where the hook is used.

Reply:
You are right I need at least the ref current to be passed to the function

Reply:
removed the function from the hook parameters so now it is defined inside the hook so this hook is expecting a children array to calculate its content height by iterating over the number of visible items that the user of the hook has specified. So if the number of visibleItems changes the useEffect will execute the heightRecalculator with the reference